monitoring/localRun.ts

```diff
@@ -1,6 +1,44 @@
+import { prompt } from 'inquirer';
+import type { CustomScheduleEventContent } from './src';
+
+type LocalRunCLIUserInput = {
+	stage: string;
+	jurisdictions: string[];
+};
+
 async function main() {
 	const { handler } = await import('./src');
-	handler().catch(console.error);
+
+	await prompt([
+		{
+			type: 'list',
+			name: 'stage',
+			message: 'Which environment would you like to test?',
+			choices: ['code', 'prod'],
+		},
+		{
+			type: 'checkbox',
+			name: 'jurisdictions',
+			message: 'Which jurisdiction would you like to test?',
+			choices: ['aus', 'ccpa', 'tcfv2'],
+			validate(input: string[]) {
+				// console.log('OI', input);
+				if (input.length === 0) {
+					return 'Please select a jurisdiction';
+				}
+				return true;
+			},
+		},
+	]).then(async (userInput: LocalRunCLIUserInput) => {
+		for (const jurisdiction of userInput.jurisdictions) {
+			const event: CustomScheduleEventContent = {
+				stage: userInput.stage,
+				jurisdiction: jurisdiction,
+			};
+			await handler(event);
```

Comment:
As we're treating `localRun` like a CLI tool, if the test fails it would make sense to fail with a non-zero exit code if the test run fails.